src/buildblock/Scanner.cxx

```diff
@@ -1231,6 +1231,64 @@ Scanner::Scanner(Type scanner_type)
       );
       break;
 
+
+    case PSMR_3rings_scanner:
+      set_params(PSMR_3rings_scanner,
+                 string_list("PSMR_3rings_scanner"),
+                 3 * 14 * 6,// + 8,
+                 411,
+                 401,
+                 24 * 7 * 5,
+                 405.0 - 18/2,
+                 7.0F,
+                 2.85F,
+                 2.08626F,
+                 static_cast<float>(-0.137), //-0.18
+                 14,//            int num_axial_blocks_per_bucket_v,
+                 5,  //            int num_transaxial_blocks_per_bucket_v,
+                 6, //            int num_axial_crystals_per_block_v,
+                 7,//            int num_transaxial_crystals_per_block_v,
+                 84, //            int num_axial_crystals_per_singles_unit_v,
+                 35,  //            int num_transaxial_crystals_per_singles_unit_v,
+                 1,
+                 0.0F, 511.F,
+#ifdef STIR_TOF
+                 1,
+                 2011.5F*2.F, // Size of coincidence window
+                 0.F
+#endif
+                 );
+      break;
+
+  case PSMR_3rings_scanner_TOF:
+    set_params(PSMR_3rings_scanner_TOF,
+               string_list("PSMR_3rings_scanner_TOF"),
+               3 * 14 * 6,// + 8,
+               344, 344,
+               24 * 7 * 5,
+               405.0 - 18/2,
+               7.0F,
+               2.85F,
+               2.08626F,
+               static_cast<float>(-0.137),
+               14,//            int num_axial_blocks_per_bucket_v,
+               5,  //            int num_transaxial_blocks_per_bucket_v,
+               6, //            int num_axial_crystals_per_block_v,
+               7,//            int num_transaxial_crystals_per_block_v,
+               252, //            int num_axial_crystals_per_singles_unit_v,
+               35,  //            int num_transaxial_crystals_per_singles_unit_v,
```

Comment:
This makes `test_Scanner` fail:
```
WARNING: Scanner PSMR_3rings_scanner_TOF: inconsistent axial singles unit info:
        num_axial_crystals_per_bucket 84 should be a multiple of num_axial_crystals_per_singles_unit 252
Error. check_consistency
```
it isn't really used otherwise though. The nonTOF version has 84, 35 for this pair, which keeps `test_Scanner` happy.
```suggestion
               84, //            int num_axial_crystals_per_singles_unit_v,
               35,  //            int num_transaxial_crystals_per_singles_unit_v,
```

Reply:
When I had 252, 35 I was still getting a warning. I change this now will it affect my fan_data, though? I am very scary as it takes hours to finish....